lib/deploy/commands.rb

```diff
@@ -39,7 +39,7 @@ def create_deploy_zip_file(*files)
 
     def use_tag_in_dockerrun(repo, tag)
       shout "Changing Dockerrun.aws.json to contain latest tag"
-      command = "sed 's/<TAG>/#{tag}/' < Dockerrun.aws.json.template > Dockerrun.aws.json"
```

Comment:
what does adding the global option to the regex do and what does removed the `<` command change?